frontend/persist.lua

```diff
@@ -34,24 +33,18 @@ local codecs = {
     },
     -- dump: human readable, pretty printed, fast enough for most user cases.
     dump = {
-
         serialize = function(t, file, as_bytecode)
-            local content, err = dump(t)
-            if not content then
-                return nil, string.format("cannot serialize table %s: %s", t, err)
-            end
             local str
             if as_bytecode then
-                str, err =  load("return " .. content)
-                if not str then
+                local bytecode, err = load("return " .. dump(t))
+                if not bytecode then
                     print("cannot convert table to bytecode: %s, ignoring", err)
```

Comment:
logger.warn ? or is that just fine because it will never happen?

Reply:
It will never happen in the context of fontlist but it will happen if the table to serialize has + 65536 nested tables.
Done :+1: